src/actions/public/L1/readFinalizedWithdrawals.ts

```diff
@@ -10,28 +10,22 @@ const CONTRACT = OpStackL1Contract.OptimismPortal
 const FUNCTION_NAME = 'finalizedWithdrawals'
 
 export type ReadFinalizedWithdrawalsParameters<
-  TChain extends Chain | undefined = Chain,
-> =
-  & { withdrawalHash: MessagePassedEvent['withdrawalHash'] }
-  & L1ActionBaseType<
-    GetL2Chain<TChain>,
-    typeof CONTRACT
-  >
+  TChain extends Chain | undefined = Chain | undefined,
+  _chainId = TChain extends Chain ? TChain['id'] : number,
+> = { withdrawalHash: MessagePassedEvent['withdrawalHash']; optimismPortal: RawOrContractAddress<_chainId> }
```

Comment:
not really related, but I think "optimismPortal" is kinda confusing and wonder if we should just call it `portal`
```suggestion
> = { withdrawalHash: MessagePassedEvent['withdrawalHash']; portal: RawOrContractAddress<_chainId> }
```

Reply:
Yeah, I'm game to rename to portal across the board. It does feel confusing to have `optimism` in a name when dealing with OP stack chains that _aren't_ Optimism